### PR TITLE
Dev/ios homescreen logged in

### PR DESCRIPTION
**Pull request by dwee9**

### CODE REVIEW

File: iOS/Utils/DateHelper.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            } else if( currentHour < 16 )
+            {
+                return "Good Afternoon";
+            } else
+            {
+                return "Good Evening";
+            }
+        }
+    }
+}
```
Comment (jhawkzz) [inline iOS/Utils/DateHelper.cs]: If GetGreetingText() is needed on Android as well, would it make sense to have a MobileApp.Shared.Utils namespace and build this in shared code? I'm assuming there will be more times where we have common functions that both platforms need.

Comment (dwee9) [reply iOS/Utils/DateHelper.cs]: That is very true.  Good point!

**Comment by dwee9** [comment]
@jhawkzz I moved it to shared

**Comment by jhawkzz** [comment]
> @jhawkzz I moved it to shared

Cool thanks! :)

**Pull request merged**